libr/bin/p/bin_elf.inc

```diff
@@ -688,7 +688,7 @@ static RList* relocs(RBinFile *bf) {
 		}
 	}
 	if (bf->o) {
-		if (!(relocs = Elf_(r_bin_elf_get_relocs) (bin))) {
+		if (!(relocs = bin->relocs)) {
```

Comment:
I'd prefer to keep the access to relocations confined in `Elf_(r_bin_elf_get_relocs)` instead of having to directly access `bin->relocs`.